keyboards/angel17/rules.mk

```diff
@@ -0,0 +1,80 @@
+# MCU name
+MCU = atmega32u4
+
+# Processor frequency.
```

Comment:
Lines 4-40 here are actually completely unnecessary as QMK now defines defaults for those rules based on the defined MCU.

https://github.com/qmk/qmk_firmware/blob/ee8d9bd40ad4f5b8d23be951d652143582d42938/quantum/mcu_selection.mk#L39-L75

See: #6253